usaspending_api/api_contracts/contracts/v2/autocomplete/recipient.md

```diff
@@ -60,16 +60,19 @@ This route sends a request to the backend to retrieve recipients matching the sp
                         "recipient_name": "ABC Holdings Inc.",
                         "recipient_level": "P",
                         "uei": "ABCDEF12345"
+                        "duns": null
```

Comment:
For one of these, could you include an example DUNS? Like `123456789'

Reply:
Yes I have updated the response with proper examples from our data.